scripts/run_celery.sh

```diff
@@ -2,27 +2,21 @@
 
 # runs celery with all celery queues except the throtted sms queue
 
-init()
-{
-    # Wait for cwagent to become available.
+set -e
 
+# Check and see if this is running in K8s and if so, wait for cloudwatch agent
+if [[ -z "${STATSD_HOST}" ]]; then
+    echo "Initializing... Waiting for CWAgent to become ready."
     while :
     do
         if  nc -vz $STATSD_HOST 25888; then
-            echo "CWAgent is Ready."
+            echo "CWAgent is Ready." > /dev/stderr
```

Comment:
Sending to std err, not std out?

Reply:
My mistake, this was from some testing I was doing. I've removed it.